frontend/src/sections/EventVolunteers/EventVolunteers.tsx

```diff
@@ -21,15 +24,35 @@ const EventVolunteers: React.FC<EventVolunteersProps> = ({ volunteers = [] }) =>
   if (volunteers.length === 0) {
     return (
       <div className='flex flex-col items-left justify-left py-10'>
-        <h4 className='text-2xl font-bold'>Event Volunteers</h4>
+        <div className='flex items-center'>
+          <h4 className='text-2xl font-bold'>Event Volunteers</h4>
+          <Link href={process.env.EVENT_VOLUNTEER_FORM || '#'} target='_blank' passHref>
+            <Button className='ml-2 group transition-all linear overflow-hidden'>
+              <span className='w-0 group-hover:w-auto overflow-hidden whitespace-nowrap transition-all linear'>
+                Become a volunteer
+              </span>
+              <FaArrowRight className='ml-0 group-hover:ml-2 transition-all linear' />
+            </Button>
+          </Link>
+        </div>
         <p className='text-gray-500 mt-4'>No volunteers are available at the moment.</p>
       </div>
     );
   }
 
   return (
     <div className='flex flex-col gap-8 py-10'>
-      <h4 className='text-2xl font-bold'>Event Volunteers</h4>
+      <div className='flex items-center'>
+          <h4 className='text-2xl font-bold'>Event Volunteers</h4>
+          <Link href={process.env.EVENT_VOLUNTEER_FORM || '#'} target='_blank' passHref>
```

Comment:
What are we trying to acheive here ? These should come from constants not env

Reply:
To prevent pushing code to prod if the form's link is changed.